### ISSUE TITLE
Themes

### ISSUE THREAD

**Issue by jimmyxander** [erming/shout#90]
Add a dark and light theme. This is a low priority but makes a big difference for UX. Being able to switch from night to day mode is very helpful on the eyes when you work at a computer all day.

I think it's important to keep it simple though and not add TONS of themes or the ability to custom theme.  I'd suggest creating a day theme and a night theme that fit the branding, the style guidelines of SHOUT itself. That will always feel more polished and careful.

**Comment by eby** [comment]
+1 for ability to change theme. I run this in public mode with multiple users so would be nice to have it stick in preferences cookie like the rest of preferences.

**Comment by YaManicKill** [comment]
I think this ticket needs clarified.

We now have themes, so I guess this is for the theme selector? Which will hopefully be fixed by #454 or #386. I assume there's nothing else to this story anymore?

**Comment by eby** [comment]
Yes those two tickets do what I needed and I'm guessing solve the original issue.
